virtcontainers/sandbox.go

```diff
@@ -1619,19 +1622,23 @@ func (s *Sandbox) updateResources() error {
 		},
 	}
 
+	var mCPU uint32
+
+	// Calculate running total of memory and mCPUs requested
 	for _, c := range s.config.Containers {
 		if m := c.Resources.Memory; m != nil && m.Limit != nil {
 			*sumResources.Memory.Limit += *m.Limit
 		}
 		if cpu := c.Resources.CPU; cpu != nil {
 			if cpu.Period != nil && cpu.Quota != nil {
-				*sumResources.CPU.Period += *cpu.Period
-				*sumResources.CPU.Quota += *cpu.Quota
+				mCPU += uint32(utils.CalculatemCPUs(*cpu.Quota, *cpu.Period))
 			}
+
 		}
 	}
 
-	sandboxVCPUs := uint32(utils.ConstraintsToVCPUs(*sumResources.CPU.Quota, *sumResources.CPU.Period))
+	//For a sandbox, we need to take the ceiling when converting from mCPU to CPU
+	sandboxVCPUs := (mCPU + 99) / 100
```

Comment:
Since the math is based on 100, could you please add a helper function for it in utils.go just like `utils.CalculatemCPUs` that is also basing on 100 (milli-CPUs).

Reply:
A function for going from milli to vCPU?  Sure.